src/routes/homepage-redesign-preview/index.svelte

```diff
@@ -1,106 +1,260 @@
+    >
+  </div> -->
+  <div class="cta-card" slot="cta-card">
+    <Card
+      {...{
+        titleText: "Cal-Adapt is evolving!",
+        linkPath: `/help/`,
+        description:
+          "Learn about the Cal-Adapt enterprise and our mission to support California's climate change initiatives and preview our future plans.",
+        height: 18,
+        ctaText: "Read More",
+      }}
+    >
+      <div class="icon-circle bg-teal-60" slot="icon_slot">
+        <svelte:component this="{InformationFilled32}" fill="red" />
+      </div>
+    </Card>
+  </div>
+</Banner>
 
 <div class="spacing--v-32"></div>
 
-<Grid>
-  <Row>
-    <Column lg="{{ span: 12 }}">
+<div class="bx--grid">
+  <div class="bx--row">
+    <div class="bx--col-lg-12 bx--col-md-8 bx--col-sm-4">
+      <!-- Fifth Assessment Notification -->
+      <div class="alert">
+        <InlineNotification
+          lowContrast
+          hideCloseButton
+          kind="info"
+          title="Looking for climate data for California's Fifth Climate Change Assessment?"
+        >
+          <p class="bx--inline-notification__subtitle">
+            Visit the <a href="/data">Cal-Adapt Analytics Engine</a>
+          </p>
+        </InlineNotification>
+      </div>
+
+      <!-- Fourth Assessment Block -->
+      <h2>Explore interactive maps and charts</h2>
+      <div class="spacing--v-24"></div>
+      <p class="full-width">
+        Visualize and download <strong>downscaled CMIP5 climate data</strong>
+        and other datasets developed for California’s
+        <a href="/blog/slr-cis-tool" target="_blank"
+          >Fourth Climate Change Assessment.</a
+        >
+        Read our <a href="/get-started" target="_blank">Get Started</a> guide to
+        learn more about working with climate data.
+      </p>
+      <p class="full-width">
+        Designed for a <strong>broad range of users.</strong>
+      </p>
+      <div class="spacing--v-24"></div>
       <CardsContainer gridGap="{2}" cardWidth="{cardWidth}">
-        {#each cardsData as card}
+        {#each cardsData as cardDatum, index}
           <Card
             {...{
-              ...card,
+              ...cardDatum,
               height: cardHeight,
+              ctaText: "Learn More",
+              textColor: "white",
               useRule: true,
-              textColor: "var(--white)",
             }}
           />
         {/each}
       </CardsContainer>
-    </Column>
 
-    <Column lg="{{ span: 3, offset: 1 }}">
-      <h2 class="h4">Latest Updates</h2>
-      <p>
-        Learn more about the latest changes and improvements to Cal-Adapt on our
-        blog.
+      <!-- Analytics / Fifth Assessment Block -->
+      <div class="spacing--v-24"></div>
+      <h2>Analyze next generation climate data</h2>
+      <div class="spacing--v-24"></div>
+
+      <div class="bx--grid">
+        <div class="bx--row">
+          <div
+            class="bx--col-lg-4 bg-gradient-analytics-engine ae-logo-box lift"
```

Comment:
The cards in the previous 4th Assessment block use CSS Grid to place the cards. The height of each card is hard coded to `18rem`. To align the AE card and AE text, you might need to use CSS Grid instead of the Carbon Grid.

Reply:
@melijimenez11 After thinking some more, if you use CSS grid to layout the AE blocks, maybe you won't need any nested grid/row/cols